src/main/groovy/org/shipkit/internal/gradle/GitPlugin.java

```diff
@@ -154,6 +142,25 @@ public void execute(final Exec t) {
         });
     }
 
+    private List<String> getAddCommand(List<String> files) {
```

Comment:
It's not that easy in this case, since we add list of files into the list with other arguments.